week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -1,98 +1,233 @@
+  "https://imagery.pragprog.com/products/595/tpp20.jpg?1557173963"]
+function insertBooks(books) {
+ 
+//   //Write your code in here
+//  get the content id by using getElementsById method
+  let container = document.getElementById('content');
+// //creating unOrdered list
+  let unOrderList = document.createElement('ul');
+  unOrderList.style.display = 'flex';
+//   //iterating our each property in books object
+  //for (let item in books) {
+  for (let item = 0; item < books.length; item++) {
+    
+    //     //creating list items for each property.
+    let listItem = document.createElement('li');
+    listItem.style.margin = '10px';
+    listItem.style.height = '250px';
+    listItem.style.listStyle = 'none';
+    let bookParagraph = document.createElement('p');
+    let bookDescription = document.createTextNode(books[item].title + ' : ' + books[item].author + '.');
```

Comment:
Consider using a template literal for this one - is good practice for the sytax. Here is a [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)